jwst/pathloss/path_loss.py

```diff
@@ -93,7 +99,7 @@ def calculate_pathloss_vector(pathloss_refdata, pathloss_wcs, xcenter, ycenter):
         crval1 = pathloss_wcs.crval1
         cdelt1 = pathloss_wcs.cdelt1
         for i in np.arange(wavesize):
-            wavelength[i] = crval1 +(float(i+1) - crpix1)*cdelt1
+            wavelength[i] = crval1 +(float(i) - crpix1)*cdelt1
```

Comment:
Since CRPIX is 1-indexed doesn't that also then require 1-indexing of the pixel numbers? So why the reversal here from 1-indexed to 0-indexed?